src/test/java/seedu/address/logic/commands/todolistcommands/FindTaskCommandTest.java

```diff
@@ -0,0 +1,184 @@
+package seedu.address.logic.commands.todolistcommands;
+
+// import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.commons.core.Messages.MESSAGE_TASKS_LISTED_OVERVIEW;
+// import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.tag.Tag;
+import seedu.address.model.task.Date;
+import seedu.address.model.task.FindTaskCriteria;
+import seedu.address.model.task.Priority;
+import seedu.address.model.task.TaskContainsTagsPredicate;
+import seedu.address.model.task.TaskMatchesDatePredicate;
+import seedu.address.model.task.TaskMatchesPriorityPredicate;
+import seedu.address.model.task.TaskNameContainsKeywordsPredicate;
+import seedu.address.model.util.SampleDataUtil;
+import seedu.address.testutil.FindTaskCriteriaBuilder;
+
+public class FindTaskCommandTest {
+
+    //private Model model = new ModelManager(new ModuleList(), new ArchivedModuleList(),
+    //        new ContactList(), getTypicalTodoList(), new EventList(), new UserPrefs());
+    //private Model expectedModel = new ModelManager(new ModuleList(), new ArchivedModuleList(),
+    //        new ContactList(), getTypicalTodoList(), new EventList(), new UserPrefs());
```

Comment:
Is this going to be used later on? If not we can remove.

Reply:
Yes this will be used, I am waiting for Michael to implement his todo builder class before I can implement junit tests for this class.